phoenix-tracing-webapp/pom.xml

```diff
@@ -155,6 +131,7 @@
         <plugin>
           <groupId>com.github.searls</groupId>
           <artifactId>jasmine-maven-plugin</artifactId>
+          <version>2.2</version>
```

Comment:
You are overriding the version set in the master pom pluginManagement section.
I suggest that you override the plugin version there instead.